### DISCUSSION TITLE
Inflect component attributes when using the OpenAPI SchemaGenerator

### DISCUSSION THREAD

**Discussion by mattis-rosenkranz** [django-json-api/django-rest-framework-json-api#966] [Ideas]
The generated OpenaAPI-Scheme component attributes are not inflected using the settings described in the docs.
This is not a problem as the parser just leaves the attribute names as they are, but it would be nice to conform the OpenAPI-scheme to the chosen field format.

**Comment by sliverc** [comment]
Thanks for reporting this. I agree this is a bug and should be fixed. I've created issue #967. I lock this discussion not to have two separate conversations about the same issue. Any more details should be added to #967 directly.
